clients/src/main/java/org/apache/kafka/common/requests/DeleteTopicsResponse.java

```diff
@@ -53,8 +53,14 @@
      */
     private static final Schema DELETE_TOPICS_RESPONSE_V2 = DELETE_TOPICS_RESPONSE_V1;
 
+    /**
+     * v3 request is the same that as v2. The response is different based on the request version.
+     */
+    private static final Schema DELETE_TOPICS_RESPONSE_V3 = DELETE_TOPICS_RESPONSE_V2;
+
     public static Schema[] schemaVersions() {
-        return new Schema[]{DELETE_TOPICS_RESPONSE_V0, DELETE_TOPICS_RESPONSE_V1, DELETE_TOPICS_RESPONSE_V2};
+        return new Schema[]{DELETE_TOPICS_RESPONSE_V0, DELETE_TOPICS_RESPONSE_V1,
+            DELETE_TOPICS_RESPONSE_V2, DELETE_TOPICS_RESPONSE_V3};
     }
 
 
```

Comment:
Can we add the new error codes (TOPIC_DELETION_DISABLED,NVALID_REQUEST)  to the below comment?

Reply:
Done!